create_table.lua

```diff
@@ -193,9 +198,26 @@ for _, tblname in ipairs(tables) do
         end
         local tuple = box.tuple.new(t)
 
-        box.space[tblname:upper()]:insert(tuple)
+        local rc, msg = pcall(box.space[sql_table].insert, box.space[sql_table], tuple)
+        if not rc then
+            print(('database %s failed to insert tuple %s: %s'):
+                format(sql_table, require'json'.encode(tuple), msg))
+        end
     end
     f:close()
+    box.commit()
+    -- sanity check - assume we have successfully inserted all tuples
+    local check_lines = box.execute(('select count(*) from %s'):format(sql_table))
+    if not check_lines then
+        error(('database %s is not existing'):format(sql_table))
```

Comment:
The error message looks rather strange. Can we change it to `database %s does not exist`?